game-app/game-core/src/test/java/games/strategy/ui/UtilTest.java

```diff
@@ -8,35 +8,37 @@
 
 import java.awt.Polygon;
 import java.lang.reflect.InvocationTargetException;
+import java.util.concurrent.atomic.AtomicReference;
 import javax.swing.SwingUtilities;
 import org.junit.jupiter.api.Nested;
 import org.junit.jupiter.api.Test;
 
 final class UtilTest {
-  Throwable inThreadException;
+  AtomicReference<Throwable> inThreadException;
```

Comment:
Now this can be a local variable instead of a field

Reply:
... or removed completely when using `invokeAndWait` directly and not in a separate thread :)
I could clean out a lot with this hint. Thanks!